.github/workflows/upload-pypi-source.yml

```diff
@@ -129,6 +129,10 @@ jobs:
             command: flake8 benchmarks build_scripts chia tests tools *.py
           - name: pylint
             command: pylint benchmarks build_scripts chia tests tools *.py
+          - name: generated protocol tests
+            command: |
+              python3 -m tests.util.build_network_protocol_files
+              git diff --exit-code
```

Comment:
Note that this will at least not catch any new files if they are ever created.

Reply:
I think that's ok for now

Reply:
actually. it won't catch new files that failed to be added to the repo. If the files *were* added, and the differ from the generated ones, it would be caught